packages/schemas/src/consts/oidc.ts

```diff
@@ -58,6 +58,10 @@ export enum ExtraParamsKey {
    * @see {@link SignInIdentifier} for available values.
    */
   Identifier = 'identifier',
+  /**
+   * The one-time token used as a proof for the user's identity. Example use case: Magic link.
+   */
+  Token = 'token',
```

Comment:
1. should we specify a format for future use like Google One Tap?
2. thinking if `token` is too general, maybe some better name? not sure though

Reply:
1. Will Google One Tap re-use the token sign-in flow?
2. If `token` is too general, how about `one-time-token`? Or do you have any suggestions?

Reply:
Discussed offline and aligned usage.

Use the search param pattern: `?one_time_token=token_value`